src/data/actions/coupons.js

```diff
@@ -63,7 +63,7 @@ const fetchCouponOrders = options => (
         logError(error);
         // This endpoint returns a 404 if no data exists,
         // so we convert it to an empty response here.
-        if (error.response.status === 404) {
+        if (error.response?.status === 404) {
```

Comment:
? is to handle null, right? What do we do here if we don't get a response at all? Is there a higher level timeout?

Reply:
This will dispatch the `fetchCouponOrdersFailure` action if there's no response. Mostly this change is for my dev environment which was throwing an uncaught error when the server was down instead of dispatching the failure action. But now that I think of it, I'm not sure what it is that got returned that *had* a response on it but not a status. I'll throw another `?` in there just in case.

Reply:
Really this fix is for me in development so that I don't get a blank page when my backend isn't running. We are not doing higher level error handling, afaik. (ie, the class does not do any error handling). I can add an extra question mark after error so that if we actually get *nothing* back, it doesn't throw.